sig-service-catalog/charter.md

```diff
@@ -5,28 +5,28 @@ README](https://github.com/kubernetes/community/blob/master/committee-steering/g
 
 ## Scope
 
-Service Catalog is a Kubernetes extension project that implements the [Open
-Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI). It enables
-application developers to provision cloud services from within Kubernetes and
-integrates configuration and discovery of those services into Kubernetes
-resources.
+Service Catalog is a Kubernetes extension project that implements and provides
+specifications for kubernetes-specific solutions that enable application developers to
+consume cloud-provided or self-hosted services from within Kubernetes and integrates
+configuration and discovery of those services into Kubernetes resources.
 
 ### In scope
 
 See the [service-catalog SIG entry](https://github.com/kubernetes/community/tree/master/sig-service-catalog).
 
-This SIG’s main goals are:
-- Support, and adhere to, the Platform requirements of the [OSBAPI
-  specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md).
-- Provide a UX for Kubernetes users that is consistent with both the OSB API
-  specification and traditional Kubernetes user interactions.
-- Align with the OSBAPI specification as changes are made.
-- Provide feedback (bugs or feature requests) to the [OSBAPI WG]](https://www.openservicebrokerapi.org/).
+Areas of focus:
+
+- Develop specifications for solutions related to configuring, discovering and connecting cloud services to applications running on Kubernetes
+- Host implementations of the abovementioned specifications
+- Provide a UX for Kubernetes users that is consistent with traditional Kubernetes user interactions.
 
 ### Code, Binaries and services
 
-- [Source Repository](https://github.com/kubernetes-sigs/service-catalog)
-  - See [OWNERS](https://raw.githubusercontent.com/kubernetes-sigs/service-catalog/master/OWNERS) for who has access.
+- Source repositories
+  - [kubernetes-sigs/service-catalog](https://github.com/kubernetes-sigs/service-catalog)
+  - [kubernetes-sigs/go-open-service-broker-client](https://github.com/kubernetes-sigs/go-open-service-broker-client)
+  - [kubernetes-sigs/minibroker](https://github.com/kubernetes-sigs/minibroker)
```

Comment:
adding this muddies "Operation of OSBAPI Service Brokers" as being out of scope

Reply:
Makes sense. I will remove that sentence.